tests/python/unittest/test_operator.py

```diff
@@ -367,6 +367,108 @@ def test_pow_fn():
     check_symbolic_forward(y, [x], [2**x])
     check_symbolic_backward(y, [x], [np.ones(shape)], [np.log(2) * 2**x])
 
+def test_equal():
```

Comment:
In "make lint" I see several failures in unrelated files, how should I go about this? Should I fix only the files which I have touched in this change ?

=====43/51 python files passed check=====
python/mxnet/io.py: 1 Errors of 1 Categories map={'refactor': 1}
python/mxnet/model.py: 1 Errors of 1 Categories map={'convention': 1}
python/mxnet/symbol.py: 1 Errors of 1 Categories map={'convention': 1}
python/mxnet/visualization.py: 1 Errors of 1 Categories map={'error': 1}
plugin/opencv/__init__.py: 1 Errors of 1 Categories map={'convention': 1}
python/mxnet/notebook/callback.py: 1 Errors of 1 Categories map={'convention': 1}
python/mxnet/image.py: 1 Errors of 1 Categories map={'refactor': 1}
plugin/opencv/opencv.py: 7 Errors of 2 Categories map={'error': 4, 'convention': 3}
38 files failed lint